SeleniumDemoQA/SeleniumDemoQA/Tests/WebTableTests.cs

```diff
@@ -0,0 +1,64 @@
+﻿using SeleniumDemoQA.Pages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeleniumDemoQA.Tests
+{
+    public class WebTableTests: BaseClass
+    {
+        [Test]
+        public void VerifyAddNewRowTable()
+        {
+            var webTablePage = new WebTablePage(_driver);
+            webTablePage.Open();
+
+            int initialRowCount = webTablePage.GetDataRowsCount();
+
+            webTablePage.AddButton();
+            webTablePage.FillFirstNameInput("Elchin");
+            webTablePage.FillLastNameInput("Sangu");
+            webTablePage.FillEmailInput("elchin@ask.com");
+            webTablePage.FillAgeInput("41");
+            webTablePage.FillSalaryInput("50000");
+            webTablePage.FillDepartmentInput("Finance");
+            webTablePage.SubmitButton();
+
+
+            int expectedRowCount = webTablePage.GetDataRowsCount();
+            //Assert.That(expectedRowCount, Is.EqualTo(initialRowCount + 1), "The row count has not incresed in the web table");
```

Comment:
why this is commented? It does not work?

Reply:
Yes, and i don't understand, why..! Because in Test below, it worked... But, after I have changed the name, as you recommend, it has worked.. something magical for me..;))